packages/cli/src/lib/collect/command-object.spec.ts

```diff
@@ -30,18 +30,18 @@ const cli = (args: string[]) =>
     ...DEFAULT_CLI_CONFIGURATION,
     commands: [yargsCollectCommandObject()],
   });
-let logs = [];
+let logs;
 
 describe('collect-command-object', () => {
   beforeEach(() => {
+    logs = [];
```

Comment:
I would also scope this variable to the `describe` block and add some type annotation (otherwise it's implicitly `any`).

```suggestion
describe('collect-command-object', () => {
  let logs: unknown[];

  beforeEach(() => {
    logs = [];
```

Reply:
Problem is, in some tests this is done in some `setup` and `teardown` function which uses this variable.... but in those cases where it isn't, I can do that yes.